services/nutanix-ai/2.1.0/helmreleases/nai-ui-dashboard-cm.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nai-ui
+  namespace: ${workspaceNamespace}
+  labels:
+    "kommander.d2iq.io/application": "nutanix-ai"
+  annotations:
+    "kustomize.toolkit.fluxcd.io/ssa": "IfNotPresent"
```

Comment:
I don't know the context around having this here - but wouldn't `Merge` (if not `Override`) be a better startegy here? Otherwise 2.1.0 NAI configmap will point to 2.0.0 `docsLink` etc., ?